fbpic/lpa_utils/laser/laser_profiles.py

```diff
@@ -926,7 +925,7 @@ def E_field( self, x, y, z, t ):
         env = self.interp_function(x, y, (t-self.t_start) )
         # Add laser oscillations
         E = (env * np.exp(
-            -1.j*self.omega * (t - self.t_start + self.t_min_lasy)
```

Comment:
hey just to remind @RemiLehe, that i'm not sure I agree with that -- this  would mess up the phase that in lasy is defined on its own time axis

Reply:
Ah, yes. Do you mean that it should simply be:
```
(t - self.t_start)
```

Reply:
oh, I think my comment outdated -- there was a change with `self.t_min_lasy` being removed, but now it's reverted. false alarm =)